tests/Wallabag/CoreBundle/Helper/ContentProxyTest.php

```diff
@@ -684,7 +684,7 @@ public function testWebsiteWithValidUTF8TitleDoNothing()
         // You can use https://www.online-toolz.com/tools/text-hex-convertor.php to convert UTF-8 text <=> hex
         // See http://graphemica.com for more info about the characters
         // '😻ℤz' (U+1F63B or F09F98BB; U+2124 or E284A4; U+007A or 7A) in hexadecimal and UTF-8
-        $actualTitle = $this->hexToStr('F09F98BB' . 'E284A4' . '7A');
```

Comment:
I think we can keep the concatenation stuff because it's how the comment explain it.

Reply:
will cancel the PHP-CS-Fixer responsible for that then